src/main/java/ch/njol/skript/expressions/ExprArmorSlot.java

```diff
@@ -52,7 +52,7 @@ public class ExprArmorSlot extends PropertyExpression<LivingEntity, Slot> {
 		if (Material.getMaterial("WOLF_ARMOR") != null)
 			bodyEntities.add(Wolf.class);
 
-		register(ExprArmorSlot.class, Slot.class, "(%-*equipmentslots%|[the] armo[u]r)[s] [item:item[s]]", "livingentities");
+		register(ExprArmorSlot.class, Slot.class, "(%-*equipmentslots%|[the] armo[u]r) [item:item[s]]", "livingentities");
```

Comment:
```suggestion
		register(ExprArmorSlot.class, Slot.class, "(%-*equipmentslots%|[the] armo[u]r[s]) [item:item[s]]", "livingentities");
```
otherwise it's breaking